invenio_rdm_records/ext.py

```diff
@@ -75,7 +80,31 @@ def verify_token():
                 # we need to add the LinkNeed here
                 if hasattr(g, "identity"):
                     g.identity.provides.add(LinkNeed(data["id"]))
-
+            else:
```

Comment:
Note: in Zenodo token data is provided in the `token=` args of URL. In RDM it's "reserved" for SecretLinks. 

Suggestion/question: should we force users to put something like `resource-access-token=` to the URL args, of this `else` here is fine?

Reply:
The name of the query string parameter could be configurable. The issue we currently have in Zenodo is that it is "hard-coded" and used by legacy clients, in which case we need to consider that it could collide with secret link tokens. So the `else` is unfortunately necessary for now. 

We might be able to mitigate this though since we're passing everything via the legacy REST API layer in ZenodoRDM, and thus we could perform the parameter "rename" on that level... The issue is that this handler runs in `before_request`, so there's little room to interject.

Reply:
What do you think if I make both RAT url arg and SecretLink url arg configurable? Or is it not possible because can be used by legacy clients?

Reply:
Nevermind... discussed irl, decided to make separate `resource-access-token=` url arg and move it to config to avoid hard-code